src/neo/SmartContract/NNS/NNSContract.Policy.cs

```diff
@@ -0,0 +1,132 @@
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using Neo.VM.Types;
+using System;
+using System.Linq;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.NNS
+{
+    public partial class NNSContract
+    {
+        public override string ServiceName => "Neo.Native.NNS";
+        public override int Id => -5;
+        public override string Name => "NNS";
+        public override string Symbol => "nns";
+
+        public override byte Decimals => 1;
+
+        protected const byte Prefix_Root = 22;
+        protected const byte Prefix_Record = 24;
+        protected const byte Prefix_Admin = 26;
+        protected const byte Prefix_RentalPrice = 27;
+        protected const byte Prefix_ReceiptAddress = 28;
+
+        public NNSContract()
+        {
+        }
+
+        internal override bool Initialize(ApplicationEngine engine)
+        {
+            if (!base.Initialize(engine)) return false;
+
+            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_Root), new StorageItem
+            {
+                Value = new UInt256[0].ToByteArray()
```

Comment:
UInt256 ? Are we using `UInt256` to represent a domain name? It'll waste a lot of space.